cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -491,21 +491,6 @@ define([
             AjaxHelpers.expectNoRequests(requests);
         });
 
-        it('should have appropriate class names on focus/blur', function (done) {
```

Comment:
They are claiming that it's an issue with Firefox. We can detect which browser is being tested with and conditionally run the test using a tool like [bowser](https://www.npmjs.com/package/bowser).